inc/spbc-settings.php

```diff
@@ -2324,6 +2407,67 @@ function spbc_field_traffic_control_logs__prepare_data(&$table)
     }
 }
 
+/**
+ * Creates a banner with a notification
+ * @param string $flag_text_banner which banner text to show
+ * @return void
+ */
+function spbc_settings__create_notice_on_tab($flag_text_banner)
+{
+    global $spbc;
+
+    switch ($flag_text_banner) {
+        case 'found_critical_files': // Creates a banner with a notification if important files are found and if the banner has not already been closed
+            if (!array_key_exists('spbct_notice-found_critical_files', $_COOKIE)) {
+                $_COOKIE['spbct_notice-found_critical_files'] = '0';
+            }
+
+            if ($spbc->data['display_scanner_warnings']['critical'] > 0 &&
+                $_COOKIE['spbct_notice-found_critical_files'] != '1') {
+                $email = spbc_get_admin_email();
+                $website = get_home_url();
+                $text = __("There's a high probability that your website has been compromised, as critical files show signs of infection. Take action now by ordering malware removal from our experienced security specialists.", 'security-malware-firewall');
+
+                //generate button
+                $landing_page_link = 'https://l.cleantalk.org/website-malware-removal?email=' . esc_attr($email) . '&website=' . esc_attr($website);
+                $button_text = __('Request Malware removal', 'security-malware-firewall');
+                $button_div = '<div style="align-content: center;margin: 0 30px;">';
+                $button_div .= '
+                    <a class="spbc_manual_link" target="_blank" href="' . $landing_page_link . '">'
+                    . '<i class="spbc-icon-link-ext"></i>&nbsp;&nbsp;'
+                    . $button_text
+                    . '</a>
+                    ';
+                $button_div .= '</div>';
+                $text .= $button_div;
```

Comment:
Is it possible to use the new [`LinkConstructor`](https://github.com/CleanTalk/security-malware-firewall/pull/389/files#diff-22b5810a774313389a32bb2f031865f7c0a2de007eb6ea04e1d0cfa4f7aec812) class?

Reply:
It can be done, but when the class is in the dev branch